classes/main.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'app'
+
+class TempMain
+  def self.display_options
+    puts 'Please choose an option by entering a number:
+    1 - List all books.
+    2 - List all people.
+    3 - Create a person (teacher or student, not a plain Person).
+    4 - Create a book.
+    5 - Create a rental.
```

Comment:
When I tried to create a rental, it broke the app:

<img width="707" alt="Screen Shot 2022-08-26 at 21 37 17" src="https://user-images.githubusercontent.com/69011963/186953204-9143268c-1054-4153-85ed-8accd8ea5de2.png">

- [ ] Make sure all errors are handled!